<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Test_2q5</class>
 <widget class="QMainWindow" name="Test_2q5">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>706</width>
      <height>180</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="backButton_for_q4">
         <property name="text">
          <string>Назад</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; font-style:italic;&quot;&gt;Вопрос 5:&lt;/span&gt;&lt;span style=&quot; font-family:'Trebuchet MS,Georgia,Bitstream Charter,serif'; font-size:16pt; font-weight:600; font-style:italic; color:#333333; background-color:transparent;&quot;&gt;К вам пришли без&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Trebuchet MS,Georgia,Bitstream Charter,serif'; font-size:16pt; font-weight:600; font-style:italic; color:#333333; background-color:transparent;&quot;&gt;предупреждения и не вовремя. &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Trebuchet MS,Georgia,Bitstream Charter,serif'; font-size:16pt; font-weight:600; font-style:italic; color:#333333; background-color:transparent;&quot;&gt;Как вы себя поведете?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="radioButton_15">
         <property name="text">
          <string>Постараюсь, чтобы гость не заметил, что его приход нежелателен</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_25">
         <property name="text">
          <string> Воскликну: «Как я рада!»</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_35">
         <property name="text">
          <string>Не попытаюсь скрыть того, что я не в восторге от его прихода</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>208</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="closeButton_for_Begin">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>805</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
